### ISSUE TITLE
Not working on Next

### ISSUE THREAD

**Issue by Ray-P** [heidkaemper/tailwindcss-intersect#2]
Hello,

Very usefull lib but it's not working with NextJs & Tailwind.

Package.json dependencies
```json
"next": "12.2",
"react": "^18.2.0", 
```
Package.json devDependencies
```json
"tailwindcss-intersect": "^1.0.1"
```
tailwind.config.js file:
```
{
  {...}
  plugins: [require("tailwindcss-intersect")]
}
```

Code example:

```html
<h2 className="text-white transition text-5xl uppercase intersect:text-red">Tailwindcss-intersect ❤️</h2>
```

The _"no-intersect"_ attribute is not **added** or **removed**.
![image](https://user-images.githubusercontent.com/6727063/192566657-640cac98-27eb-4ebe-9cdf-8ef59e91773c.png)

Maybe your `Observer script` is crashing because document is not defined yet (server rendering mode) ?
```js
document.addEventListener("DOMContentLoaded", () => {...}
```

Thank you @heidkaemper  :)

**Comment by heidkaemper** [comment]
Thanks for reporting this. I'll have a look.

**Comment by heidkaemper** [comment]
Thanks again for your input and sorry for the long wait.

The problem is hopefully fixed with the 1.0.2 release:
https://github.com/heidkaemper/tailwindcss-intersect/releases/tag/1.0.2

With this version this plugin is working for me with Next.js. In the useEffect hook for instance.

If there is still a problem, please let me know.

**Comment by Ray-P** [comment]
I will check it asap 😊
